src/main/java/org/eclipse/tractusx/ssi/lib/jwt/SignedJwtFactory.java

```diff
@@ -116,10 +117,16 @@ private static SignedJWT createSignedES256Jwt(
 
       var algorithm = JWSAlgorithm.EdDSA;
       var type = JOSEObjectType.JWT;
+
+      // https://w3c.github.io/vc-jws-2020/#json-web-signature-2020
```

Comment:
This only applies to detached JWS, I'm not convinced it applies in this context.

Reply:
Yes the VP and the VC both expect a JsonWebSignature2020 in their proof, where the `jws` attribute is always detached. This fix shouldn't have any negative impact on the VP creation.